dlt/__init__.py

```diff
@@ -79,6 +80,7 @@
     "TCredentials",
     "sources",
     "destinations",
+    "dataset",
```

Comment:
let's export `_dataset` until we have stable interface.

Reply:
done